src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -110,12 +120,58 @@ public void deleteTags() {
         assertEquals(expectedPerson, actualPerson);
     }
 
+    @Test
+    public void getGroups_returnsImmutableSet() {
```

Comment:
Perhaps name of test method can be getGroups_validGroupInput_returnsImmutableSet()?